### ISSUE TITLE
Missing git binary

### ISSUE THREAD

**Issue by pinkwah** [flathub/com.gitbutler.gitbutler#67]
This is a bit silly, but this flatpak doesn't seem to ship with a `git` binary. How should I use this program?

**Comment by honnip** [comment]
No, this package isn't available without modification, so I didn't think it would be a problem (if someone is going to modify the package anyway, they would probably bind the git also!)

Anyway, I'll work on this package for a while

**Comment by NN708** [comment]
Resolved in a577eb217224ecc7a3f764fa3b53e610023c4039.
